src/Server/Options/ServerOptions.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+namespace ZfrOAuth2\Server\Options;
+
+use Zend\Stdlib\AbstractOptions;
+
+
+/**
+ * Options class
+ *
+ * @author  Michaël Gallego <mic.gallego@gmail.com>
+ * @licence MIT
+ */
+class ServerOptions extends AbstractOptions
+{
+    /**
+     * Object manager key
+     *
+     * @var string|null
+     */
+    protected $objectManager;
```

Comment:
I think so, though the name suggests an instance while it is in fact something like 'orm_default'. The manager registry takes null or a named service manager.

> Op 26 jan. 2016 om 00:04 heeft Michaël Gallego <notifications@github.com> het volgende geschreven:
> 
> In src/Server/Options/ServerOptions.php:
> 
> > +
> > +
> > +/**
> > + * Options class
> > + *
> > + * @author  Michaël Gallego <mic.gallego@gmail.com>
> > + * @licence MIT
> > + */
> > +class ServerOptions extends AbstractOptions
> > +{
> > +    /**
> > +     * Object manager key
> > +     *
> > +     * @var string|null
> > +     */
> > +    protected $objectManager;
> Is this needed if we use ManagerRegistry?
> 
> —
> Reply to this email directly or view it on GitHub.
>